myuw/static/vue/components/_templates/boilerplate/search.vue

```diff
@@ -19,18 +19,15 @@
               placeholder="Search the UW website"
             />
             <b-input-group-append>
-              <b-button
-                variant="purple"
-                class="rounded-0"
-                type="submit"
+              <button class="btn btn-purple rounded-0" type="submit"
```

Comment:
do we have btn-purple defined?

Reply:
we do generate btn-purple.
Here are all our custom variants
https://github.com/uw-it-aca/myuw/blob/72071cfb03c4605ec3e0174de0874ec6a2ab53d3/myuw/static/css/myuw/variables.scss#L15-L48